test/client/desktop/IPCTest.js

```diff
@@ -162,6 +163,7 @@ o.spec("IPC tests", function () {
 			utilsMock: n.mock("../api/common/utils/Utils", utils).set(),
 			autoUpdaterMock: n.mock("__updater", autoUpdater).set(),
 			alarmSchedulerMock: n.mock("__alarmScheduler", alarmScheduler).set(),
+			themeManagerMock: n.mock("__themeManagre", themeManager).set(),
```

Comment:
typo

Reply:
We should get rid of that argument already